build.sbt

```diff
@@ -574,9 +576,10 @@ lazy val unit = project
       "io.get-coursier" %% "coursier" % V.coursier, // for jars
       "ch.epfl.scala" %% "bloop-config" % V.bloop,
       "org.scalameta" %% "munit" % V.munit,
-      // Only here to have scala-steward update V.ammonite.
-      // Not actually used in tests.
-      "com.lihaoyi" %% "ammonite-util" % V.ammonite intransitive ()
+      // The dependencies listed below are only listed so Scala Steward
+      // will pick them up and update them. They aren't actually used.
+      "com.lihaoyi" %% "ammonite-util" % V.ammonite intransitive (),
+      "com.lihaoyi" % "mill-contrib-testng" % V.mill intransitive ()
```

Comment:
The package name seems odd, but it is indeed just `testng`. I figured this was a nice small one to include for this purpose.